content/release-notes/1.52.0.md

```diff
@@ -9,6 +9,10 @@ kubernetes: "1.19, 1.20, and 1.21"
  * Added the ability to force [image garbage collection](/kots-cli/admin-console/garbage-collect-images/) via the KOTS CLI even if rollback is enabled for the application by passing the `--ignore-rollback` flag.
 {{</features>}}
 
+{{<changes>}}
+ * KOTS can now successfully [install](/kots-cli/install/) and [upgrade](/kots-cli/admin-console/upgrade/) when the `readOnlyRootFilesystem` is required in an existing cluster.
```

Comment:
@ajp-io @samantha-replicated Yeah, I agree `the` doesn't make sense. I was intentionally vague about what it meant and how to enforce it, because this setting doesn't need to be explicit and there is more than one way to enforce. How about the following:

```suggestion
 * KOTS can now successfully [install](/kots-cli/install/) and [upgrade](/kots-cli/admin-console/upgrade/) when [`readOnlyRootFilesystem`](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) is enforced in an existing cluster.
```

Reply:
Agh, ok that's context is helpful. I'm good with your suggestion and committed it. Thanks for clarifying!